### PR TITLE
Add a rake task to print a table of time-since-last-release info

### PR DESCRIPTION
**Pull request by mfurtak**
Run with `rake release_stats` from the repo root

screen shot 2016-04-19 at 12 53 27 pm

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,6 +1,8 @@
 GEMS = %w(fastlane fastlane_core deliver snapshot frameit pem sigh produce cert gym pilot credentials_manager spaceship scan supply watchbuild match screengrab)
 RAILS = %w(boarding refresher enhancer)
 
+SECONDS_PER_DAY = 86_400
```
Comment (KrauseFx) [inline Rakefile]: Maybe just use `60 * 60 * 24` here?

File: rakelib/release_stats.rake [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+RED_DAY_COUNT = 14
+
+def should_row_be_red?(row)
+  row[1] > RED_COMMIT_COUNT || (row[2] > RED_DAY_COUNT && row[1] > 0)
+end
+
+def colorize_rows(rows)
+  rows.map do |row|
+    if should_row_be_red?(row)
+      row.map { |item| item.to_s.red }
```
Comment (KrauseFx) [inline rakelib/release_stats.rake]: So Ruby ❤️

**Comment by KrauseFx** [comment]
Looks great to me 👍

**Comment by asfalcone** [comment]
LGTM 👍

**Pull request merged**